src/java/picard/analysis/artifacts/ContextAccumulator.java

```diff
@@ -47,7 +50,9 @@ public void countRecord(final String refContext, final char calledBase, final SA
         final ListMap<Transition, DetailPair> detailMetricsMap = new ListMap<Transition, DetailPair>();
         for (final Transition altTransition : Transition.altValues()) {
             final Transition refTransition = altTransition.matchingRef();
-            for (final String context : this.artifactMap.get(altTransition).keySet()) {
+            final TreeSet<String> orderedContexts = new TreeSet<String>();
```

Comment:
All three new lines could be replaced with: `for (final String context : new TreeSet<>(this.artifactMap.get(altTransition).keySet()) {`